dockerfiles/cfme_tests_base/Dockerfile

```diff
@@ -1,12 +0,0 @@
-# Our custom docker image with app
```

Comment:
I think it can be compatible. It would be nice to have just one base image instead of two. I would like to leverage automated builds feature on DockerHub. How do we build PRT images right now? Manually?